source/initial-dns-seedlist-discovery/tests/README.rst

```diff
@@ -11,42 +11,27 @@ Test Setup
 Start a three-node replica set on localhost, on ports 27017, 27018, and 27019,
 with replica set name "repl0".
 
-There are two methods to provide the SRV records required for testing: either
-mock them with the resolv_wrapper library and test only on Linux, or configure
-the SRV records with a real name server.
-
-If you choose the first method, build the `CWRAP resolv_wrapper library
-<https://cwrap.org/resolv_wrapper.html>`_ and initialize its hosts file, for
-example with this shell script:
-
-.. code-block:: sh
-
-  if [ ! -f resolv_wrapper_build/src/libresolv_wrapper.so ]; then
-    curl -LO https://ftp.samba.org/pub/cwrap/resolv_wrapper-1.1.5.tar.gz
-    tar xvf resolv_wrapper-1.1.5.tar.gz
-    mkdir resolv_wrapper_build
-    cd resolv_wrapper_build
-    cmake ../resolv_wrapper-1.1.5
-    make
-    cd ..
-  fi
-
-  cat << EOF > hosts.txt
-  SRV _mongodb._tcp.test1.test.build.10gen.cc localhost 27017
-  SRV _mongodb._tcp.test1.test.build.10gen.cc localhost 27018
-  SRV _mongodb._tcp.test2.test.build.10gen.cc localhost 27018
-  SRV _mongodb._tcp.test2.test.build.10gen.cc localhost 27019
-  SRV _mongodb._tcp.test3.test.build.10gen.cc localhost 27017
-  EOF
-
-  export LD_PRELOAD=`pwd`/resolv_wrapper_build/src/libresolv_wrapper.so
-  export RESOLV_WRAPPER_HOSTS=hosts.txt
-
-You can debug the library by exporting ``RESOLV_WRAPPER_DEBUGLEVEL=3``.
-
-If you choose instead to configure a real name server with SRV records, adapt
-the records shown above to your domain name, and update the "uri" field in the
-YAML or JSON files in this directory with the actual domain.
+To run the tests that accompany this spec, you need to configure the SRV and
```

Comment:
We'll need to inform OFFICEIT to also add a TXT record to accompany the SRV record they set up for us

Reply:
Yeah, I already wrote to drivers leads about that. Don't want to do that before this is approved though (which you can do by going to https://github.com/mongodb/specifications/pull/194/files and click the "Review Changes" button).

Reply:
@derickr yep I know how to approve :) I'm waiting for feedback on the tests below

Reply:
Open a BUILD ticket to add TXT records, not OFFICEIT.